include/rmm/device_scalar.hpp

```diff
@@ -46,12 +46,8 @@ class device_scalar {
       T const &initial_value, cudaStream_t stream_ = 0,
```

Comment:
```suggestion
      T const &initial_value, cudaStream_t stream = 0,
```

Reply:
How come you removed the trailing `_` from `stream` and not `rm`?

Also, how come github is showing that a removed space from the beginning of the line? It looks like there are still 6 spaces.

Reply:
Who knows. I missed the mr.